src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  filterBy: string;
+  setFilterBy: (filter: string) => void;
+}
+
+export const TodoFilter: React.FC<Props> = ({ filterBy, setFilterBy }) => {
+  return (
+    <nav className="filter">
+      <a
+        href="#/"
```

Comment:
Use 'e.preventDefault()' to prevent the default behavior of the anchor tag when it's clicked.